.travis.yml

```diff
@@ -18,7 +18,7 @@ before_install:
     - export CXX="g++-4.8"
 
 before_script:
-    - npm install juttle@^0.5.0;
+    - npm install juttle@^0.6.0;
```

Comment:
why not `npm install juttle` ? pulls the latest and greatest... thats what I've done in the graphite one.